ij-core/src/main/kotlin/org/jetbrains/research/ij/headless/server/utils/PsiFileUtils.kt

```diff
@@ -20,3 +23,13 @@ fun createPsiFile(project: Project, fileName: String, language: Language, text:
     return PsiFileFactory.getInstance(project).createFileFromText(fileName, language, text)
         ?: error("Can not create file $fileName")
 }
+
+fun findPsiFileByName(project: Project, fileName: String): PsiFile {
+    val scope = GlobalSearchScope.allScope(project)
+    
+    val mainVirtualFile = FilenameIndex.getVirtualFilesByName(fileName, scope).firstOrNull() ?: run {
+        error("Can not find the $fileName file in the ${project.name} project")
+    }
+
+    return mainVirtualFile.toPsiFile(project)!!
```

Comment:
I'd prefer to throw an error if we cannot convert the virtual file into a psi one